content/general/concepts/api/api.md

```diff
@@ -27,3 +27,21 @@ APIs can be divided into three groups:
 - Private APIs, also known as Internal APIs, are used primarily within a company to share resources and facilitate the business. (e.g. Company Warehouse API for managing inventory with code).
 
 - Partner APIs require rights or specific licenses for use. These APIs are popular in software-as-a-service platforms (e.g. [AWS API](https://docs.aws.amazon.com/general/latest/gr/aws-apis.html)).
+
+## Benefits and Risks of Using APIs
+
+The main benefits of using APIs are:
+
+- Cost savings through the elimination of the need to build and maintain local physical infrastructure and services.
+
+- Opportunities to leverage the expertise of other vendors instead of having to reinvent solutions for common problems. (authentication, payment processing, maps, etc.)
```

Comment:
Can we make "(authentication, payment processing, maps, etc.)" its own sentence by explicitly stating these are some examples, or if we want to keep it in parenthesis we should move it inside the sentence?

Reply:
Should be inside the sentence, I guess...